docs/open-sdg-features.md

```diff
@@ -30,6 +43,18 @@ It is possible to "boost" one or more fields in the search index, so that more r
 
 It is also possible to index additional metadata fields, so that more fields are searched e.g. if you wanted the search to return pages where the graph title contains the search term. See the [search_index_extra_fields configuration option](https://open-sdg.readthedocs.io/en/latest/configuration/#search_index_extra_fields) for how to configure this.
 
+## Accessibility High Contrast version
+As well as the default contrast version, Open SDG also offers a high contrast version. By default two menu buttons show, to allow users to choose between the different contrast levels, for example on the [US site](https://sdg.data.gov/). Another approach is to use a contrast toggle button, for example on the [UK site](https://sustainabledevelopment-uk.github.io/).
+
+For guidance on how to use the more accessible contrast button, see the Configuration page [contrast_type](configuration.md) section.
+
+## Multilingual
+By default, the site is created in English but this can be changed and multiple languages can be added.
```

Comment:
Technically there is not a "default" language. The site-starter does start off with English (and Spanish too I think) but that is just the way that the site-starter is configured. I might recommend this language here: `The Open SDG platform can be either monolingual or multilingual. All aspects of the platform are included in the translation system, including page content, indicator metadata, indicator data disaggregations, menus and interface elements, etc.`